tests/python/unittest/test_optimizer.py

```diff
@@ -94,7 +94,9 @@ def update(self, index, weight, grad, state):
 
         t = self._index_update_count[index]
         mean, variance = state
-        grad *= self.rescale_grad
+
```

Comment:
@leezu 
Please feel free to put in the same change to https://github.com/dmlc/mxnet/blob/master/perl-package/AI-MXNet/t/optimizers.t#L77